### ISSUE TITLE
Error when trying to load resource

### ISSUE THREAD

**Issue by mikeller** [betaflight/betaflight-configurator#949]
When running on linux, with language set to 'default', the following error message is shown in the console when starting up:

`Failed to load resource: net::ERR_FILE_NOT_FOUND _locales/en-GB/messages.json`

Looks like the operating system defaults to `en-GB` as language, and we need to shorten this to `en` (or potentially do a prefix match).

**Comment by McGiverGim** [comment]
If I don't remember bad, the i18next does it automatically, if not found fallbacks to shorten form and maybe is only a warning. I will try to verify it today.
In the worst case, the fallback to English is done always and the language can be directly selected from the options menu.
